python/foglamp/common/plugin_discovery.py

```diff
@@ -10,6 +10,8 @@
 from foglamp.common import logger
 from foglamp.services.core.api import utils
 from foglamp.services.core.api.plugins import common
+from foglamp.plugins.common import utils as api_utils
```

Comment:
This should be as common_utils. 

We already have `from foglamp.services.core.api import utils` above